### PR TITLE
chore(containerization): add deploy frontend.yml

### PR DESCRIPTION
**Pull request by mkholjuraev**
This PR is intended to create [deployment config file](https://github.com/RedHatInsights/patchman-ui/pull/832). Some of config info is taken from: https://github.com/RedHatInsights/cloud-services-config/blob/ci-beta/chrome/fed-modules.json#L291.

### CODE REVIEW

**Comment by mkholjuraev** [comment]
@BlakeHolifield thank you for super quck review. I will fix all issues.

File: deploy/frontend.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      title: Patch
+      deploymentRepo: https://github.com/RedHatInsights/patchman-ui
+      API:
+        versions:
+          - v1
+      frontend:
+        paths:
+          - /insights/patch
+      image: ${IMAGE}:${IMAGE_TAG}
+      navItems:
```
[the remark this answers is not in the document]
Comment (mkholjuraev) [reply deploy/frontend.yml:21]: I have no idea why, but I can not ask another review. I would like mark this convesation resolved to check if this blocking.

[the remark this answers is not in the document]
Comment (mkholjuraev) [reply deploy/frontend.yml:21]: Nope did not help.

**Comment by mkholjuraev** [comment]
@BlakeHolifield can you please have another look whenever you are free?

File: deploy/frontend.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+              title: 'Advisories'
+              href: '/insights/patch/advisories'
+            - appId: 'patch'
+              title: 'Systems'
+              href: '/insights/patch/systems'
+            - appId: 'patch'
+              title: 'Packages'
+              href: '/insights/patch/packages'
+            - appId: 'patch'
+              title: 'Patch Template'
```
Comment (BlakeHolifield) [inline deploy/frontend.yml:38]: Should this be template or patch-set? https://github.com/RedHatInsights/cloud-services-config/blob/ci-beta/chrome/rhel-navigation.json#L72 . I want to make sure I'm looking at the right source file for the nav.

Comment (mkholjuraev) [reply deploy/frontend.yml:38]: Initially we were using Patch Set, but recently UX requested to rename to 'Patch template'. I have a ready PR to rename all titles on Patch. I wanted to rename the nav also directly with this PR.

**Review by BlakeHolifield — APPROVED** [review]

**Pull request merged**